GVRf/Framework/jni/engine/memory/gl_delete.h

```diff
@@ -25,15 +28,48 @@
 #define GVR_INVALID 0
 
 namespace gvr {
+
+extern pthread_key_t deleter_key;
+
 class GlDelete {
 
 public:
+    /**
+     * Before using this class this method must be called once
+     * and only once per-process
+     */
+    static void createTlsKey() {
+        //corresponding pthread_key_delete missing intentionally as this is
+        //needed for as long as the process is alive
+        int err = pthread_key_create(&deleter_key, nullptr);
```

Comment:
This function references an internal data "deleter_key". it can be moved to gl_deleter.cpp.